commons/src/test/java/info/metadude/android/eventfahrplan/commons/temporal/MomentTest.kt

```diff
@@ -105,4 +105,48 @@ class MomentTest {
         assertThat(moment2.isBefore(moment1)).isFalse()
         assertThat(moment1.isBefore(moment1)).isFalse()
     }
+
+    @Test
+    fun plusSeconds() {
+        val moment = Moment(0)
+        moment.plusSeconds(1)
+
+        assertThat(moment.toMilliseconds()).isEqualTo(1000)
+
+        moment.plusSeconds(-1)
+        assertThat(moment.toMilliseconds()).isEqualTo(0)
+    }
+
+    @Test
+    fun plusMinutes() {
+        val moment = Moment(0)
+        moment.plusMinutes(1)
+
+        assertThat(moment.toMilliseconds()).isEqualTo(1000 * 60)
+
+        moment.plusMinutes(-1)
+        assertThat(moment.toMilliseconds()).isEqualTo(0)
+    }
+
+    @Test
+    fun minusHours() {
+        val moment = Moment(3600 * 1000)
+        moment.minusHours(1)
+
+        assertThat(moment.toMilliseconds()).isEqualTo(0)
+
+        moment.minusHours(-1)
+        assertThat(moment.toMilliseconds()).isEqualTo(3600 * 1000)
+    }
+
+    @Test
+    fun minusMinutes() {
+        val moment = Moment(60 * 1000)
+        moment.minusMinutes(1)
+
+        assertThat(moment.toMilliseconds()).isEqualTo(0)
+
+        moment.minusMinutes(-1)
+        assertThat(moment.toMilliseconds()).isEqualTo(60 * 1000)
+    }
```

Comment:
Thank you for writing all these unit tests! It provides a warm and cozy feeling. :+1: